src/gui/addnewtorrentdialog.cpp

```diff
@@ -587,6 +597,11 @@ void AddNewTorrentDialog::accept()
     // Label
     params.label = ui->label_combo->currentText();
 
+    if (ui->default_label->isChecked()) {
+        pref->setSaveLabel(params.label);
+        pref->apply();
```

Comment:
No need to call Preferences::apply() here, because this value is only meaningful for AddNewTorrentDialog.